Wrappers/Python/cil/framework/__init__.py

```diff
@@ -16,20 +16,12 @@
 # Authors:
 # CIL Developers, listed at: https://github.com/TomographicImaging/CIL/blob/master/NOTICE.txt
 
-import numpy
-import sys
-from datetime import timedelta, datetime
-import warnings
-from functools import reduce
-from .framework import cilacc
-from .framework import DataContainer
-from .framework import ImageData, AcquisitionData
-from .framework import ImageGeometry, AcquisitionGeometry
-from .framework import VectorData, VectorGeometry
-from .framework import find_key, message
-from .framework import DataProcessor, Processor
-from .framework import AX, PixelByPixelDataProcessor, CastDataContainer
-from .block import BlockDataContainer
-from .block import BlockGeometry
-from .framework import DataOrder
-from .framework import Partitioner
+from .cilacc import cilacc
```

Comment:
Shouldn't be exposed.

Reply:
If has to be exposed, since all the usages are `from cil.framework import cilacc`.
If you remove this line, all the usages must be changed to `from cil.framework.cilacc import cilacc`